README.md

```diff
@@ -9,7 +9,7 @@
 
 [Demo Screencast]
 
-`function` is a Client Library and CLI for enabling the development of implicitly deployed, platform agnostic code.
+`func` is a Client Library and CLI for enabling the development of implicitly deployed, platform agnostic code.
```

Comment:
unrelated, but it is really a _Client Library_ ?

Reply:
🤷

Reply:
I can remove that if you want to.